aeolus/aeolus.h

```diff
@@ -46,92 +45,93 @@ class M_ifc_init;
 //   Synth
 //---------------------------------------------------------
 
-class Aeolus : public Synthesizer {
-      static const std::vector<ParDescr> pd;
+class Aeolus : public Synthesizer
+{
+    static const std::vector<ParDescr> pd;
 
-      Model* model;
-      QList<MidiPatch*> patchList;
-      uint16_t _midimap [16];
-      int _sc_cmode;          // stop control command mode
-      int _sc_group;          // stop control group number
+    Model* model;
+    QList<MidiPatch*> patchList;
+    uint16_t _midimap[16];
+    int _sc_cmode;            // stop control command mode
+    int _sc_group;            // stop control group number
 
-      enum { VOLUME, REVSIZE, REVTIME, STPOSIT };
+    enum {
+        VOLUME, REVSIZE, REVTIME, STPOSIT
+    };
 
-      volatile bool   _running;
-      int             _hold;
-      int             _nplay;
-      int             _nasect;
-      int             _ndivis;
-      Asection*       _asectp [NASECT];
+    volatile bool _running;
+    int _hold;
+    int _nplay;
+    int _nasect;
+    int _ndivis;
```

Comment:
whitespace for alignment is still destroyed here

Reply:
It becomes not aligned 15 lines below (in the old codebase). I would not align the data members. And definitely not in the aeolus :)

Reply:
This file was just a random example, I guess the same applies to many other files too
Actually the aeolus stuff should probably be excluded from the reformatting at all, being basically third party code.
But seems that bird has flown meanwhile